ajax/libs/highlighter.js/0.1.8/highlighter.min.js

```diff
@@ -0,0 +1,12 @@
+//# sourceMappingURL=highlighter.min.js.map
```

Comment:
Because there is no the map file in dist on npm.
I think I should open an issue to ask author, right?